config/linux-nitropad-x.config

```diff
@@ -299,7 +299,7 @@ CONFIG_BOOT_VESA_SUPPORT=y
 CONFIG_NR_CPUS_RANGE_BEGIN=2
 CONFIG_NR_CPUS_RANGE_END=512
 CONFIG_NR_CPUS_DEFAULT=64
-CONFIG_NR_CPUS=64
+CONFIG_NR_CPUS=32
```

Comment:
@daringer Only change i'm not sure about

Reply:
should be fine